### ISSUE TITLE
Safe Robot Learning in Assistive Devices through Neural Network Repair. (arXiv:2303.04431v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#25085]
Safe Robot Learning in Assistive Devices through Neural Network Repair. (arXiv:2303.04431v1 [cs.RO])<br>
https://ift.tt/XQGIPx0<br>
<p>Assistive robotic devices are a particularly promising field of application for neural networks (NN) due to the need for personalization and hard-to-model human-machine interaction dynamics. However, NN based estimators and controllers may produce potentially unsafe outputs over previously unseen data points. In this paper, we introduce an algorithm for updating NN control policies to satisfy a given set of formal safety constraints, while also optimizing the original loss function. Given a set of mixed-integer linear constraints, we define the NN repair problem as a Mixed Integer Quadratic Program (MIQP). In extensive experiments, we demonstrate the efficacy of our repair method in generating safe policies for a lower-leg prosthesis.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
